examples/resources/spacelift_stack/resource.tf

```diff
@@ -42,7 +42,7 @@ resource "spacelift_stack" "k8s-cluster-bitbucket-datacenter" {
   terraform_version = "0.12.6"
 }
 
-# Terraform stack using GitHub Enterprise as VCS
+# Terraform stack using [GitHub custom application](https://docs.spacelift.io/integrations/source-control/github#setting-up-the-custom-application) as VCS
```

Comment:
Will this actually render correctly? Won't it just show the code since this is inside a code-block?

If so, maybe we should just do something like:

```markdown
# Terraform stack using a GitHub Custom Application. See the following page for more info: https://docs.spacelift.io/integrations/source-control/github#setting-up-the-custom-application
```

Reply:
That is a good point! Will change this. Thanks!

Reply:
Done. Would you mind reviewing again @adamconnelly , please?